src/api/browserview/browserview.ts

```diff
@@ -81,11 +82,11 @@ export class BrowserView extends WebContents<BaseEventMap> {
         const ack = await this.wire.sendAction('get-browser-view-info', {...this.identity});
         return ack.payload.data;
     }
-
     /**
     * Retrieves the window the view is currently attached to.
     * @experimental
-    * @return {Promise.<_Window>}
+    * @return {Promise.<
```

Comment:
Invalid JSDoc

Reply:
Good catch, must have happened in the conflict resolution